root.c

```diff
@@ -448,6 +448,20 @@ luaA_root_size(lua_State *L)
     return 2;
 }
 
+/** Get the physical size of the root window, in millimeter.
+ *
+ * @return Width of the root window, in millimeters.
+ * @return height of the root window, in millimeters.
+ * @function size_mm
+ */
+static int
+luaA_root_size_mm(lua_State *L)
+{
+    lua_pushinteger(L, globalconf.screen->width_in_millimeters);
+    lua_pushinteger(L, globalconf.screen->height_in_millimeters);
+    return 2;
```

Comment:
Scratch that, we actually get the new physical size via RRChangeNotifyEvent, we just need to keep track of the update.